storage/innobase/handler/ha_innodb.cc

```diff
@@ -11989,6 +12018,11 @@ ha_innobase::update_create_info(
 	    && create_info->tablespace == NULL) {
 		create_info->tablespace = m_prebuilt->table->tablespace;
 	}
+	if ((m_prebuilt->table->flags2 & DICT_TF2_ENCRYPTION)
+			&& create_info->encrypt_type.length == 0) {
+		static const LEX_STRING yes = { C_STRING_WITH_LEN("Y") };
+		create_info->encrypt_type = yes;
```

Comment:
Use of stack variable after scope? Just in case, please grep, such LEX_STRING/LEX_CSTRING might be already available elsewhere for reuse

Reply:
I don't see a stack variable? I only see a static const.
But it would also work with a stack variable, because the assignment calls the copy constructor, and the struct only contains a const char pointer to a string literal, and an integer (the length)

Reply:
Right, sorry